tests/test_power.py

```diff
@@ -0,0 +1,72 @@
+"""Tests for the _ATEN_PE6108G PDU class"""
+import time
+
+import mock
+from nose.tools import eq_, raises
+from pysnmp.entity.rfc3413.oneliner import cmdgen
+from pysnmp.proto.rfc1902 import Integer
+
+from _stbt.power import uri_to_power_outlet
+
+
+def mock_data(int_value):
+    """Match the format of the data returned from pysnmp"""
+    return (None, None, None, [(oid, Integer(int_value))])
+
+
+cmdgen.UdpTransportTarget = mock.Mock()
+time.sleep = mock.Mock()
+
+outlet = 1
+oid = "1.3.6.1.4.1.21317.1.3.2.2.2.2.{0}.0".format(outlet + 1)
+aten = uri_to_power_outlet('aten:mock.host.name:1')
+
+
+@mock.patch('pysnmp.entity.rfc3413.oneliner.cmdgen.CommandGenerator')
+def test_aten_get_on(mock_class):
+    mock_class.return_value.getCmd.return_value = mock_data(2)
+
+    result = aten.get()
+
+    eq_(result, True)
```

Comment:
Please use `assert results is True` here.  I find `assert` statements more readable, and the advantages of `eq_` will be negated once we move to py.test with it's [assert inspection](http://pytest.org/latest/assert.html#assert-with-the-assert-statement).